src/__tests__/home.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import * as router from 'react-router';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../redux/configureStore';
+import Home from '../Pages/Home';
+import Details from '../Pages/Details';
+
+describe('Ensure app renders as expected', () => {
+  const navigate = jest.fn();
+
+  beforeEach(() => {
+    jest.spyOn(router, 'useNavigate').mockImplementation(() => navigate);
+  });
+  test('renders the home page', () => {
+    render(<Provider store={store}><Home /></Provider>);
+    expect(screen.findAllByText('Cryptocurrency Trading Data')).toMatchSnapshot();
+  });
+  test('renders the Details page', () => {
+    render(<Provider store={store}><Details /></Provider>);
```

Comment:
- [ ] Please Consider adding to your Application more unit tests for all components of the app and also test some pure functions in Redux code. 👨🏻‍🎨